release/models/system/openconfig-system-logging.yang

```diff
@@ -426,6 +432,12 @@ module openconfig-system-logging {
         "Source IP address for packets to the log server";
     }
 
+    leaf network-instance {
+      type oc-ni:network-instance-ref;
+      description
+        "The network instance used to find the log server.";
```

Comment:
Should we specify the behavior if the network-instance isn't configured? (i.e. what is the assumption today without this leaf? The default NI?).